__tests__/app.test.js

```diff
@@ -200,7 +193,30 @@ describe('/api/articles/:article_id/comments', () => {
             expect(commentObj).toMatchObject({
                 comment_id: 19,
                 body: 'Cats are excellent creatures.',
-                author: 'Mochi'
+                author: 'lurker'
+            })
+        })
+    })
+    test('POST:201 ignores additional properties, inserts a new comment to the db and sends the new comment back to the client)', () => {

```

Comment:
Good test, just make sure you put in an expect statement to say it'll be a 201, this proves its working 👍